gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1540,19 +1539,14 @@ gst_tensor_buffer_get_nth_memory (GstBuffer * buffer,
     return NULL;
   }
 
-  if (!info) {
-    nns_loge ("Failed to get tensors info (invalid input info).");
-    return NULL;
-  }
-
-  if (info->num_tensors <= 0) {
-    nns_loge ("num_tensors is 0. Please check the tensors info.");
+  num_tensors = gst_buffer_n_tensor (buffer);
+  if (num_tensors <= 0) {
```

Comment:
condition (num_tensors == 0U), num_tensors is unsigned int.

Reply:
Thanks, I updated the line.